packages/flame/test/components/position_component_test.dart

```diff
@@ -404,6 +404,51 @@ void main() {
       expect(component.positionOf(Vector2(10, 0)), Vector2(44, 22));
     });
 
+    test('double center flips', () {
+      final startPosition = Vector2(50, 20);
+      final component = PositionComponent()
+        ..size = Vector2(10, 20)
+        ..position = startPosition;
+      final centerPosition = component.center;
+
+      component.flipVerticallyAroundCenter();
```

Comment:
Does it make sense to apply scale as well in test?

Reply:
That could be a good idea, added scale and rotation.